fem/tmop_amr.cpp

```diff
@@ -837,26 +837,26 @@ void TMOPHRSolver::ParUpdate()
 }
 #endif
 
-void TMOPHRSolver::UpdateNonlinearFormAndBC(Mesh *mesh, NonlinearForm *nlf)
+void TMOPHRSolver::UpdateNonlinearFormAndBC(Mesh *_mesh, NonlinearForm *_nlf)
```

Comment:
I don't want to be that picky guy, but I'd like to avoid underscores as a prefix. See https://www.gnu.org/software/libc/manual/html_node/Reserved-Names.html paragraph 2.

Reply:
Agreed, at some point we discussed this, and settled on suffix underscores (`mesh_`, `nlf_`) instead.

Reply:
That's fine. I had forgotten about that. I'll change it.